docs/api/system-preferences.md

```diff
@@ -121,6 +143,20 @@ Returns `Number` - The ID of this subscription
 Same as `subscribeNotification`, but uses `NSNotificationCenter` for local defaults.
 This is necessary for events such as `NSUserDefaultsDidChangeNotification`.
 
+**[Deprecated](modernization/promisification.md)**
+
+### `systemPreferences.subscribeLocalNotification(event)` _macOS_
+
+* `event` String
+
+Returns `Promise<Object>` - Resolves with an object containing the following items:
+  * `id` Number - The ID of this subscription
+  * `event` String
+  * `userInfo` Object
```

Comment:
Can we type this as `unknown` or if we know the type maybe a `Record<A, B>`?

Reply:
`userInfo`, you mean?

Reply:
Yeah